lib/elixir/lib/inspect/algebra.ex

```diff
@@ -54,6 +54,9 @@ defmodule Inspect.Opts do
       `:number`, `:atom`, `regex`, `:tuple`, `:map`, `:list`, and `:reset`.
       Colors can be any `t:IO.ANSI.ansidata/0` as accepted by `IO.ANSI.format/1`.
 
+    * `:inspect_fun` - a function to build algebra document, defaults to
```

Comment:
Similar to https://github.com/elixir-lang/elixir/pull/8978, @josevalim ?

```suggestion
    * `:inspect_fun` (since v1.9.0) - a function to build algebra document, defaults to
```